app/src/main/java/com/github/factotum_sdp/factotum/ui/roadbook/DRecordList.kt

```diff
@@ -0,0 +1,90 @@
+package com.github.factotum_sdp.factotum.ui.roadbook
+
+import com.github.factotum_sdp.factotum.data.DestinationRecord
+
+
+private fun displayedRecords(allRecords: List<DestinationRecord>,
```

Comment:
Hum why putting this method outside the DRecordList?

Reply:
Then yeah I think put it inside a companion object is the best idea I think !

Reply:
Yes `displayedRecords()` is directly linked to a `DRecordList` and used only by that class internally.

Ok, thank you for revealing that point.